module/documents/chat-message.mjs

```diff
@@ -324,11 +324,19 @@ export default class ChatMessage5e extends ChatMessage {
     const item = this.getAssociatedItem();
     if ( this.isContentVisible && item ) {
       const isCritical = (roll.type === "damage") && this.rolls[0]?.options?.critical;
-      const subtitle = roll.type === "damage"
-        ? isCritical ? game.i18n.localize("DND5E.CriticalHit") : game.i18n.localize("DND5E.DamageRoll")
-        : roll.type === "attack"
-          ? game.i18n.localize(`DND5E.Action${item.system.actionType.toUpperCase()}`)
-          : item.system.type?.label ?? game.i18n.localize(CONFIG.Item.typeLabels[item.type]);
+      let subtitle;
+      if ( roll.type === "damage" ) {
+        subtitle = isCritical ? game.i18n.localize("DND5E.CriticalHit") : game.i18n.localize("DND5E.DamageRoll");
+        if ( roll.damageOnSave ) subtitle = `${subtitle} (${
```

Comment:
The subtitle is not really intended for large amounts of informational text like this. This would be better as the message or roll flavour text I think.

Reply:
<img width="298" alt="Screenshot 2024-08-14 at 09 22 38" src="https://github.com/user-attachments/assets/bd275235-9c0a-461c-afa1-3738aab4e3b6">

Reply:
Clearly the entire Silmarillion you shoved in there.